Tools/Scripts/libraries/webkitscmpy/webkitscmpy/remote/bitbucket.py

```diff
@@ -40,14 +40,18 @@ class PRGenerator(Scm.PRGenerator):
         def PullRequest(self, data):
             if not data:
                 return None
+            author = self.repository.contributors.create(
+                data['author']['user']['displayName'],
+                data['author']['user'].get('emailAddress', None),
+            )
+            author.bitbucket = data['author']['user'].get('name', None)
```

Comment:
It seems we keep repeating assigning `bitbucket` to variable created by `self.repository.contributors.create`.
Any reason we don't make `create` take `bitbucket=None` argument?